### PR TITLE
[ENTESB-7532] Create new quickstart to cover camel-hl7

### PR DESCRIPTION
**Pull request by jkasztur**

### CODE REVIEW

**Comment by johnpoth** [comment]
@jkasztur  Awesome work! I'll take a look and run through the example

File: quickstarts/camel/camel-hl7/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+7. To see what is happening within the JBoss Fuse server, you can continuously view the log (tail) with the following command
+
+       log:tail
+
+File Based Test
+---------------
+
+1. As soon as the Camel route has been started, you will see a directory `work/camel-hl7/input` in your JBoss Fuse installation.
+2. Copy the files you find in this quick start's `src/main/resources/data` directory to the newly created `work/camel-hl7/input`
```
Comment (johnpoth) [inline quickstarts/camel/camel-hl7/README.md]: should be 'quickstart's'

File: quickstarts/camel/camel-hl7/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+directory. There are to example files - valid.hl7 and invalid.hl7
```
Comment (johnpoth) [inline quickstarts/camel/camel-hl7/README.md]: should be 'two'

File: quickstarts/camel/camel-hl7/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+3. If the message is valid, you will see succesful response in log:
```
Comment (johnpoth) [inline quickstarts/camel/camel-hl7/README.md]: should be 'successful'

File: quickstarts/camel/camel-hl7/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+       	at ca.uhn.hl7v2.parser.GenericParser.getVersion(GenericParser.java:191) ~[225:ca.uhn.hapi.osgi-base:2.2.0]
+       	...
+
+TCP Based Test
+--------------
+
+1. Change your working directory to `camel-hl7/src/main/resources/scripts`, you will find helper scripts to send message to TCP endpoint.
+2. To test valid file on UNIX, run:
+
+       ./send.sh ../data/valid.hl7 localhost 8888
```
Comment (johnpoth) [inline quickstarts/camel/camel-hl7/README.md]: 1- Would be nice to have a windows script to send messages
2- We usually provide an uninstall section (you can copy paste from another quickstart).

Comment (johnpoth) [reply quickstarts/camel/camel-hl7/README.md]: 3- would be nice to pipe the error message to the client?

**Comment by jkasztur** [comment]
I fixed the typos + added uninstall section.
Also tried creating a windows script, but Windows doesn't have built-in netcat (or similar program). The user would need to install additional software.
Will look into piping messages into client, but I don't think it's necessary, as everything is logged into fuse.log.

**Comment by johnpoth** [comment]
@jkasztur LGTM thanks :+1: merging

**Pull request merged**